lib/ReactViews/SelectableDimensions/SelectableDimension.tsx

```diff
@@ -44,6 +48,7 @@ const SelectableDimension: React.FC<{
       ) : null}
       {isCheckbox(dim) && <SelectableDimensionCheckbox id={id} dim={dim} />}
       {isEnum(dim) && <SelectableDimensionEnum id={id} dim={dim} />}
```

Comment:
Should this be `isSingleEnum` or `isSimpleEnum` to distinguish from `isMultiEnum`? 

Should `SelectableDimensionEnum` now be `SelectableDimensionSimpleEnum`? Just to make sure that the two are mutually exclusive going forward

Reply:
I'm not that keen on `SimpleEnum`, but I can rename `MultiEnum` to `EnumArray` or something like that

Reply:
Oh yeah I think that would clear it up nicely.